### ISSUE TITLE
RENDER WARNING: there is no texture bound to the unit (Flashing Black Regions)

### ISSUE THREAD

**Issue by kaansoral** [pixijs/pixi.js#3235]
With the latest dev build of the PIXI, I'm getting this error

Basically some textures are flashing black periodically and randomly

1 Player reported this issue to me in the past, in the way of "Flashing Black Regions" - with the latest unstable dev build, it's a regular thing for me

It happens when I open the game in a second tab, but normally, the game can be opened in 5+ tabs without any issues

Edit: Actually happens on one tab too, but feels less, It's 100% not a local or hardware issue by the way, since earlier versions work

**Comment by GoodBoyDigital** [comment]
hey @kaansoral ! This will be based on an update we recently pushed optimizing texture swaps. Can I just check, did you build the version yourself?
Cheers!

Mat

**Comment by kaansoral** [comment]
Hi

Sorry for the late reply, I tested the build on FAQ

**Comment by GoodBoyDigital** [comment]
cool, whats FAQ?

**Comment by englercj** [comment]
I think he meant he used the urls from the FAQ section of the Wiki:

https://github.com/pixijs/pixi.js/wiki/FAQs#where-can-i-get-a-build

**Comment by GoodBoyDigital** [comment]
cheers for the clarification! will take a look

**Comment by kaansoral** [comment]
I've tested the 4.1.1 build, and the latest 4.1.1 dev build, it didn't have the mentioned issue, so it was introduced and eliminated between 4.1.0-dev and 4.1.1

I think the issue the player reported could have been caused by maybe old drivers, but from the screenshots I've seen, they looked identical to the issue I've experienced recently

TL;DR: I'm glad the issue is solved, sorry for the fuss :)
